### PR TITLE
Resolves #1730

### LINKED ISSUE #1703: The documentation for the -Replace parameter is not correct.
The example -Replace @{otherTelephone='555-222-2222', '555-222-1111'} will replace any and all existing values with both specified values, "555-222-2222" any "555-222-1111".
When more than one value is to be assigned to an attribute, the values are separated by commas.
When the values for more than one attribute are being replaced, the hash entries are separated by semicolons.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 269be7ed-28d0-8f8b-be9b-9b0a53205079
* Version Independent ID: 08d4a8cc-f723-a00a-6cb1-50daee082d06
* Content: [Set-ADObject (activedirectory)](https://docs.microsoft.com/en-us/powershell/module/activedirectory/set-adobject?view=winserver2012-ps)
* Content Source: [docset/winserver2012-ps/activedirectory/Set-ADObject.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/live/docset/winserver2012-ps/activedirectory/Set-ADObject.md)
* Product: **w10**
* Technology: **powershell**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### LINKED ISSUE #1730: Make it clear that changes only take effect in *future* sessions (processes)
Also worth noting that, while a change is pending, `Get-Culture` still reports the _still effective_, not the _pending_ value (which contrasts with the behavior of `Get-WinUILanguageOverride` with respect to `Set-WinUILanguageOverride`).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 58bcd6e9-7aee-ec93-4707-3207d0166ea6
* Version Independent ID: beb3e794-eaad-4f56-175c-73f07fc19a39
* Content: [Set-Culture (international)](https://docs.microsoft.com/en-us/powershell/module/international/set-culture?view=win10-ps)
* Content Source: [docset/windows/international/Set-Culture.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/live/docset/windows/international/Set-Culture.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Add note
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1730

### CODE REVIEW

File: docset/windows/international/Set-Culture.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## OUTPUTS
 
 ## NOTES
-
+In order for changes made by this cmdlet to take effect you need to open a new PowerShell session.
```
Comment (get-itips) [inline docset/windows/international/Set-Culture.md:72]: What about
_Changes made by the use of this cmdlet will take effect on subsequent PowerShell sessions._

Comment (velkovb) [reply docset/windows/international/Set-Culture.md:72]: Sounds way better :) I will update that.

**Review by dariomws — APPROVED** [review]
I like @get-itips' suggestion.

**Review by tseward — APPROVED** [review]

**Comment by e0i** [comment]
Reopening to run checks.

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by jborsecnik** [comment]
This PR is ready to merge when the build issue is resolved.

**Comment by e0i** [comment]
Should be ready now since the build issues have been resolved.

**Comment by jborsecnik** [comment]
@velkovb, update the branch, and I should be able to merge this.

**Comment by jborsecnik** [comment]
Actually, I can do the update. ...

**Pull request merged**